go-kit/metrics/provider/librato/librato.go

```diff
@@ -452,6 +452,14 @@ func (h *Histogram) SumSq() float64 {
 type Counter struct {
 	*generic.Counter
 	p *Provider
+
+	used bool
```

Comment:
A comment on this variable would be welcome. It's only because I have more context from speaking with y'all that I understand it's purpose (and why it doesn't get reset).

Reply:
:+1: Done.